Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:6
```

Comment:
Since we're not targeting lambda with this, should we make this `node:8`? Or do we want to stick with LTS? I'm personally fine with going forward to 8 since it's a stable release, but I can see the argument either way.